examples/examples.json

```diff
@@ -0,0 +1,188 @@
+{  
+    "main": {  
+        "file": {  
+            "mapbox": "misc/guide.html"
+        }
+    },
+    "categories": [  
+        {  
+            "title": "Basics",
+            "samples": [  
+                {  
+                    "title": "Add a layer",
+                    "desc": "Add one CARTO layer to your map",
+                    "file": {  
```

Comment:
what is the purpose of the file field ?

Reply:
In CARTO.js whe use this field to point to the 'leaflet' and 'gmaps' examples. Now for vector we use mapbox but other providers could be added in the future.